src/DbgHelp/DbgHelp+LOADED_IMAGE.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="LOADED_IMAGE"/> nested type.
+    /// </content>
+    public static partial class DbgHelp
+    {
+        /// <summary>
+        /// Contains information about the loaded image.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential)]
+        public struct LOADED_IMAGE
+        {
+            /// <summary>
+            /// The file name of the mapped file.
+            /// </summary>
+            [MarshalAs(UnmanagedType.LPStr)]
+            public string ModuleName;
+
+            /// <summary>
+            /// A handle to the mapped file.
+            /// </summary>
+            public IntPtr hFile;
+
+            /// <summary>
+            /// The base address of the mapped file.
+            /// </summary>
+            public IntPtr MappedAddress;
+
+            /// <summary>
+            /// A pointer to an IMAGE_NT_HEADERS structure.
```

Comment:
All of theses structures with painfully complex layout, but that's here that the lib will shine by providing them ready to consume.